### ISSUE TITLE
Detect large trees in the foreground only; exclude small and distant trees in background (forest setting)

### ISSUE THREAD

**Issue by LopezJER** [ultralytics/ultralytics#10287]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I'm training a YOLOv8 model to detect trees in a forest setting. However, I want my model to detect only large trees that are in the foreground (i.e., those that meet a minimum projected height and width) and ignore small and distant trees in the background. Most of my images have only one or two trees that meet this criterion.

For this to work, I'm thinking of:
1) annotating only those trees that meet a minimum projected width and height (e.g., 300*300 px), and
2) disabling any augmentations related to scale 

I'm hoping that the above interventions would discourage the model from detecting small and distant trees in the background. 
What do you think of this approach? Do you have any alternatives or further suggestions to meet my use case?

**Comment by LopezJER** [comment]
Ok, great! We'll first setup a baseline experiment with the initial interventions. Then, we'll tweak the confidence thresholds and loss functions as you suggested based on the model errors.

Thanks, Glenn for confirming my intuition and anticipating strategies for our fine-tuning.

We'll let the community know of our result and follow up if need be. For now, you may close this issue :)
